Salesforce Org Setup/force-app/main/default/permissionsets/TrailInsurance_Mobile.permissionset-meta.xml

```diff
@@ -1,265 +1,162 @@
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>true</editable>
-    <field>Case.Description</field>
-    <readable>true</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>true</editable>
-    <field>Case.Incident_Location_Txt__c</field>
-    <readable>true</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>true</editable>
-    <field>Case.Incident_Location__c</field>
-    <readable>true</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>false</editable>
-    <field>Case.IsClosedOnCreate</field>
-    <readable>false</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>false</editable>
-    <field>Case.IsEscalated</field>
-    <readable>false</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>true</editable>
-    <field>Case.Origin</field>
-    <readable>true</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>false</editable>
-    <field>Case.ParentId</field>
-    <readable>false</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>true</editable>
-    <field>Case.PotentialLiability__c</field>
-    <readable>true</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>false</editable>
-    <field>Case.Priority</field>
-    <readable>false</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>true</editable>
-    <field>Case.Reason</field>
-    <readable>true</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>true</editable>
-    <field>Case.Subject</field>
-    <readable>true</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>false</editable>
-    <field>Case.SuppliedCompany</field>
-    <readable>false</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>false</editable>
-    <field>Case.SuppliedEmail</field>
-    <readable>false</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>false</editable>
-    <field>Case.SuppliedName</field>
-    <readable>false</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>false</editable>
-    <field>Case.SuppliedPhone</field>
-    <readable>false</readable>
-  </fieldPermissions>
-  <fieldPermissions>
-    <editable>true</editable>
-    <field>Case.Type</field>
-    <readable>true</readable>
-  </fieldPermissions>
-  <hasActivationRequired>false</hasActivationRequired>
-  <label>TrailInsurance Mobile</label>
-  <objectPermissions>
-    <allowCreate>false</allowCreate>
-    <allowDelete>false</allowDelete>
-    <allowEdit>false</allowEdit>
-    <allowRead>true</allowRead>
-    <modifyAllRecords>false</modifyAllRecords>
-    <object>Account</object>
-    <viewAllRecords>false</viewAllRecords>
-  </objectPermissions>
+    <description>Gives users permission to access custom fields needed for the TrailInsurance Mobile app</description>
+    <fieldPermissions>
+        <editable>false</editable>
+        <field>Account.isMaster__c</field>
```

Comment:
Needs to be removed. Source deployment fails currently as the field is removed, but still referenced here.